packages/fxa-auth-server/fxa-oauth-server/lib/config.js

```diff
@@ -276,6 +276,38 @@ const conf = convict({
       env: 'FXA_OPENID_TTL',
     },
   },
+  ppid: {
+    enabled: {
+      doc: 'Whether pairwise pseudonymous identifiers (PPIDs) are enabled',
+      format: Boolean,
+      default: true,
+      env: 'PPID_ENABLED',
+    },
+    enabledClientIds: {
+      doc: 'client_ids that receive PPIDs',
+      format: Array,
+      default: [],
+      env: 'PPID_CLIENT_IDS',
+    },
+    rotatingClientIds: {
```

Comment:
For flexibility, perhaps combine `rotatingClientIds` and `rotationPeriodMS` such that the rotation period for each client is configurable and have the default if a client is not in the list be `Infinity` (no rotation).

```js
clientRotationPeriodMS: {
  '38a6b9b3a65a1873': 21600000
}
```